### ISSUE TITLE
RFE: Add parameter to register to specify translation context

### ISSUE THREAD

**Issue by atodorov** [ecometrica/django-vinaigrette#29]
Currently my .po file looks like this:

```
#: vinaigrette-deleteme.py:2
msgid "PROPOSED"
msgstr ""

#: vinaigrette-deleteme.py:3
msgid "CONFIRMED"
msgstr ""

#: vinaigrette-deleteme.py:4
msgid "DISABLED"
msgstr ""

#: vinaigrette-deleteme.py:5
msgid "NEED_UPDATE"
msgstr ""

```
As a translator I have no idea where these strings are coming from or how they might be used.

**Comment by jordiecometrica** [comment]
I assume you mean the context can be added when registering the models with vinaigrette, right? Or do you want a mechanism to add context to every database objection under consideration?

**Comment by atodorov** [comment]
I think adding the context when registering is enough for now.

**Comment by jordiecometrica** [comment]
So, what do you think?

https://github.com/ecometrica/django-vinaigrette/pull/30

**Comment by atodorov** [comment]
Should be fixed in #30 closing.
